src/hotspot/share/prims/jvmti.xml

```diff
@@ -1861,10 +1861,10 @@ jvmtiEnv *jvmti;
         permanent features of the Java platform.</i>
         <p/>
         Resume all virtual threads except those already resumed.
-        Virtual threads may be suspended with
-        <functionlink id="SuspendAllVirtualThreads"></functionlink> or
-        <functionlink id="SuspendThreadList"></functionlink> or
-        <functionlink id="SuspendThread"></functionlink>.
+        Virtual threads may be resumed with
+        <functionlink id="ResumeAllVirtualThreads"></functionlink> or
+        <functionlink id="ResumeThreadList"></functionlink> or
+        <functionlink id="ResumeThread"></functionlink>.
         Virtual threads that are currently resumed do not change state.
```

Comment:
Just to add to Serguei's comment. I think it would be clearer if we move the sentence "Virtual threads that are currently resumed do not change state" to before the "Virtual threads may be suspended ..."

Reply:
I moved the sentence as @AlanBateman suggested.
I also moved the equivalent sentence in SuspendAllVirtualThreads.